app/models/task.py

```diff
@@ -1,5 +1,36 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from datetime import datetime
 from ..db import db
+from typing import Optional
 
-class Task(db.Model):
+class Task(db.Model): # Declares model Task, tells SQLAlchemy to map Task to db table
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str]
+    description: Mapped[str]
+    completed_at: Mapped[datetime] = mapped_column(nullable=True)
```

Comment:
```suggestion
    completed_at: Mapped[Optional[datetime]]
```

To set nullability for a table's column we should use `Optional[]` per the [SQLAlchemy documentation](https://docs.sqlalchemy.org/en/20/orm/quickstart.html#declare-models), which our curriculum follows as well. We do not need to add more specific typing information regarding nullability so we don't need to use `mapped_column`.

You can review [Building an API Part 3](https://learn-2.galvanize.com/cohorts/4352/blocks/1310/content_files/api-3-database-models-read/models-setup.md) to read about the `Optional` syntax too.
<img width="885" alt="image" src="https://github.com/user-attachments/assets/ce679238-b311-42d8-abd3-e22bc0840b7a" />

<img width="792" alt="image" src="https://github.com/user-attachments/assets/aad79ebc-3f10-41ac-ba9e-5e4ee829b980" />

You use `Optional[]` for the attributes below on line 12-13, we should take care to be consistent with the documentation and other lines of code we write too.

Reply:
Fixed as requested.